### PR TITLE
Add in logout functionality and notes field for profile page 

### PR DESCRIPTION
**Pull request by cassandrale179**

### CODE REVIEW

File: app/src/main/java/com/example/caregiver/ProfileInfo.java [hunk 1]
```diff
@@ -80,17 +80,20 @@ public void onCreate(Bundle savedInstanceState) {
...
-        EditText emailField = (EditText) view.findViewById(R.id.profileEmail);
+
+        currentName = preferences.getString("userName", "N/A");
         nameField.setHint(currentName);
+
+        currentEmail = preferences.getString("userEmail", "N/A");
         emailField.setHint(currentEmail);
+
+        currentNotes = preferences.getString("userNotes", "N/A");
+        Log.d("hmmm", currentNotes);
```
Comment (Sppurswani) [inline app/src/main/java/com/example/caregiver/ProfileInfo.java:94]: I think you forgot to remove this haha

File: app/src/main/java/com/example/caregiver/ProfileInfo.java [hunk 2]
```diff
@@ -251,8 +269,18 @@ public void onClick(View v) {
...
-    public static class MainActivity extends AppCompatActivity {
+    /**
+     * Function to logout
+     */
+    private View.OnClickListener logOutListener = new View.OnClickListener() {
+        @Override
+        public void onClick(View v) {
+            Intent i = new Intent(v.getContext(), Login.class);
+            startActivity(i);
+        }
```
Comment (Sppurswani) [inline app/src/main/java/com/example/caregiver/ProfileInfo.java:280]: I wonder if pressing the back button after logging out, will take the user in the app again. Were you able to test this?

File: app/src/main/java/com/example/caregiver/ProfileInfo.java [hunk 3]
```diff
@@ -251,8 +269,18 @@ public void onClick(View v) {
...
+    };
 
+    public static class MainActivity extends AppCompatActivity {
         @Override
         protected void onCreate(Bundle savedInstanceState) {
             super.onCreate(savedInstanceState);
```
Comment (Sppurswani) [inline app/src/main/java/com/example/caregiver/ProfileInfo.java:286]: What's the purpose of this snippet in ProfileInfo?

File: app/src/main/java/com/example/caregiver/ProfileInfo.java (hunk 1 above)
Comment (cassandrale179) [reply app/src/main/java/com/example/caregiver/ProfileInfo.java]: Oops, fixed.

File: app/src/main/java/com/example/caregiver/ProfileInfo.java (hunk 3 above)
Comment (cassandrale179) [reply app/src/main/java/com/example/caregiver/ProfileInfo.java]: Good point. I have no idea lol why I incude it in there. Will remove it.

File: app/src/main/java/com/example/caregiver/ProfileInfo.java (hunk 2 above)
Comment (cassandrale179) [reply app/src/main/java/com/example/caregiver/ProfileInfo.java]: LOL you are right. It goes back to the app. I add in 
```java
getActivity().finish()
```
This should prevent it to go back.

**Review by Sppurswani — APPROVED** [review]

**Pull request merged**